input/sqs.go

```diff
@@ -52,11 +53,13 @@ func (cfg *SQSConfig) fillDefaults() {
 }
 
 type SQS struct {
-	*inpututils.S3Input
+	s3Input *inpututils.S3Input
```

Comment:
out of curiosity: is there a reason for removing the embedding? Style? Other? thx

Reply:
Since `inpututils.S3Input` already implements the `baker.Input` interface, `SQS` didn't have to implement the `Stop` method to also implement it since it anonymously embbeded `S3Input`., and when the topology called `SQS.Stop`, this was in fact `S3Input.Stop`that was called.

We now do want to provide our own implementation of `Stop`, and though we could have done it without having this change I do believe it's better to leave no doubt to the reader as to which Stop method is called, that why I chose to un-embed it